frontend/src/components/EmployeeSearchGallery.tsx

```diff
@@ -0,0 +1,46 @@
+import {Employee} from "../model/Employee";
+import EmployeeCard from "./EmployeeCard";
+import {ChangeEvent, useState} from "react";
+import styled from "styled-components";
+
+type EmployeeSearchProps = {
+    employees: Employee[]
+    removeEmployee: (id: string) => void
+}
+
+export default function EmployeeSearchGallery(props: EmployeeSearchProps) {
+
+    const [searchText, setSearchText] = useState<string>("")
+
+    const filteredEmployees: Employee[] = props.employees.filter(employee => employee.name.toLowerCase().includes(searchText.toLowerCase()))
+
+    function onSearchChange(event: ChangeEvent<HTMLInputElement>) {
+        setSearchText(event.target.value)
+    }
+
+    return (
+        <div>
+            <StyledInput placeholder={"Search"} value={searchText} onChange={onSearchChange}/>
+
+            <StyledDiv>
+                {filteredEmployees.map(employee =>
+                    <EmployeeCard employee={employee} key={employee.id}
+                                  removeEmployee={props.removeEmployee}/>)}
+                {/*//Ich will später über das Dashboard mappen und nicht über Karten*/}
+            </StyledDiv>
+        </div>
+    )
+}
+
+const StyledDiv = styled.div`
+    display: flex;
+    gap: 15px;
+    padding: 50px;
+    margin-top: 10px;
+    border-top: black solid 2px;
+`;
```

Comment:
Ich würde versuchen auf Abstände in der Einheit px zu verzichten, denn die sind nicht responsive zu Veränderungen der Browsergröße. Mein Tip wäre mit rem zu arbeiten, aber es gibt auch andere relative Größen